[master] Choreo Connect ARM64 Ubuntu images for Tryout (#5127)
wso2/docs-apim: en/docs/includes/deploy/cc-tryout-in-arm64-note.md

>     ??? note "Tryout in ARM64 based machine (ex. Apple M1)"

Reviewer: ```suggestion
    ??? note "Tryout in ARM64 based machine (e.g., Apple M1)"
```

>         If you are trying out in **ARM64** based machine (ex. Apple M1) update the docker image names with the following in

Reviewer: ```suggestion
        If you are trying out in **ARM64** based machine (e.g., Apple M1) update the Docker image names with the following in
```

>         - docker-compose.yaml (If you are trying out with Docker)

Reviewer: ```suggestion
        - docker-compose.yaml (if you are trying out with Docker)
```

            - `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/docker-compose.yaml`
            - `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect-with-apim/docker-compose.yaml`

>         - Kubernetes deployment.yaml (If you are trying out with Kubernetes) files.

Reviewer: ```suggestion
        - Kubernetes deployment.yaml (if you are trying out with Kubernetes)
```

            - `<CHOREO-CONNECT_HOME>/k8s-artifacts/choreo-connect/adapter-deployment.yaml`
            - `<CHOREO-CONNECT_HOME>/k8s-artifacts/choreo-connect/choreo-connect-deployment.yaml`
            - `<CHOREO-CONNECT_HOME>/k8s-artifacts/choreo-connect-with-apim/apim/apim-deployment.yaml`
            - `<CHOREO-CONNECT_HOME>/k8s-artifacts/choreo-connect-with-apim/choreo-connect/adapter-deployment.yaml`
            - `<CHOREO-CONNECT_HOME>/k8s-artifacts/choreo-connect-with-apim/choreo-connect/choreo-connect-deployment.yaml`
        
        | Existing (Default) image name      | Image name for tryout in ARM64 (ex: Apple M1) |
        |------------------------------------|-----------------------------------------------|
        | wso2/choreo-connect-adapter:1.0.0  | wso2/choreo-connect-adapter:1.0.0-ubuntu      |
        | wso2/choreo-connect-enforcer:1.0.0 | wso2/choreo-connect-enforcer:1.0.0-ubuntu     |
        | wso2/choreo-connect-router:1.0.0   | wso2/choreo-connect-router:1.0.0-ubuntu       |
        | wso2/wso2am:4.0.0-alpine           | wso2am/wso2am:4.0.0-ubuntu                    |
    
        **Note:**</br>

>         Please note that, above ubuntu based (`*-ubuntu`) images are for **tryout purpose only**, **not recommonded for a production deployment**.

Reviewer: ```suggestion
        Note that the above Ubuntu based (`*-ubuntu`) images are for **tryout purpose only**, **not recommended for a production deployment**.
```
